Clinical-Genomics/MIP
# Release/9.1.1

### This PR adds | fixes:

- Fixes MIP gender estimation

SINGULARITY_BIND isn't set when MIP parses the fastqs for gender, causing the command to not find the reference. This leads to an error and MIP reports this as a zero reads found mapping to Y. This in turn leads to MIP classifying all wgs samples with unknown gender as female. 
![image](https://user-images.githubusercontent.com/3168925/102098454-891da480-3e27-11eb-88ab-41e753ef39c2.png)

This PR sets SINGULARITY_BIND for the bwa command so that the command works as expected. 

### How to test:

- Automatic and continuous test suite

- Run the case hotalpaca in dry run mode and see that MIP classifies the sample as male

- Check that SINGULARITY_BIND has it's original value after MIP execution

### Expected outcome:
- [ ] Installation, unit and integration test suite pass
- [x] MIP classifies the sample as male
- [x] SINGULARITY_BIND has its original value 

### Review:

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
